README.md

```diff
@@ -17,7 +17,9 @@ https://github.com/juspay/haskell-incremental-build-template/blob/7432331400f279
 
 https://github.com/juspay/haskell-incremental-build-template/blob/7432331400f27924d827fbf81de5140c1fa0052c/nix/modules/flake/haskell.nix#L20-L32
 
-- Adopt the CI workflow from [ci.yaml](./.github/workflows/ci.yaml).
+- Use [Find Intermediates Base](./.github/actions/find-intermediates-base) action to find the commit hash with cached build artifacts.
```

Comment:
Why is this action called `find-intermediates-base`? What does `intermediates` mean? What does `base` mean?

Reply:
`intermediates` are the build artifacts produced by an older revision of project that will be reused by the latest revision. `base` is to denote the first commit hash that has these intermediates cached.

Reply:
I have tried to elaborate this in the actions description and in the `README` of the action

Reply:
> I have tried to elaborate this in the actions description and in the `README` of the action

This one, correct?

<img width="338" alt="image" src="https://github.com/user-attachments/assets/51669b06-0e6f-44ea-bc0e-2895eef8d4ec" />

---

Based on that description, why not `find-latest-cached-commit`?

Reply:
https://github.com/juspay/haskell-incremental-build-template/pull/5/commits/fe18470a01745f8455d84aa1dd1d0d911174f13a